enterprise/conf/locale/en/LC_MESSAGES/django.po

```diff
@@ -258,22 +262,22 @@ msgstr ""
 
 #: enterprise/admin/views.py:563
 #, python-brace-format
-msgid "{enrolled_count} user was enrolled to {enrolled_in}."
-msgid_plural "{enrolled_count} users were enrolled to {enrolled_in}."
+msgid "{enrolled_count} learner was enrolled in {enrolled_in}."
+msgid_plural "{enrolled_count} learners were enrolled in {enrolled_in}."
 msgstr[0] ""
 msgstr[1] ""
 
 #: enterprise/admin/views.py:587
 #, python-brace-format
-msgid "Enrollment of some users in {enrolled_in} failed: {user_list}"
+msgid "The following learners could not be in {enrolled_in}. {user_list}"
```

Comment:
Should this be: "The following learners could not be enrolled in {enrolled_in}: {user_list}"  ?